FOUR-22605: Participant Home Screen V2 & FOUR-22600: Stages configuration

pmPaulis opened the pull request:
## Stages Bar

Implementing custom-defined waypoints that prefill a progress bar will significantly improve user experience by providing clarity, reducing uncertainty, and enhancing time management. Clients will appreciate the transparency in tracking project or task stages, making it easier to oversee complex workflows. This feature can boost both productivity and satisfaction, ensuring that Tasks are managed smoothly and efficiently.

[PRD - Stages configuration (Progress Bar)](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3986063361)
[FIGMA - Stages configuration (Progress Bar)](https://www.figma.com/design/yMpUWMQv7jD8AYWxFsuZHl/Summer-2025?node-id=1122-16953&p=f&t=QyX230LETGsF1QfC-0)

## Participant Home Screen V2
Participants in new projects often face difficulty navigating the home screen due to limited customization options. The current layout may not display the most relevant information, leading to low engagement, and lack of visibility into key metrics.

[PRD - Summer 25 Participant Home Screen V2: TCE the revenge last](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3991797761/Summer+25+Participant+Home+Screen+V2+TCE+the+revenge+last)
[
FIGMA -Summer 25 Participant Home Screen V2: TCE the revenge last](https://www.figma.com/design/yMpUWMQv7jD8AYWxFsuZHl/Summer-2025?node-id=3914-19979&m=dev)


## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-22605
- https://processmaker.atlassian.net/browse/FOUR-22600

## PR's Related
- https://github.com/ProcessMaker/screen-builder/pull/1816

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.


ci:TCE_CUSTOMIZATION_ENABLED=true

henryjonathanquispe committed:
FOUR-23575:S2: Case tab - Implement the component to show a counters

gproly committed:
[FOUR-23473](https://processmaker.atlassian.net/browse/FOUR-23473) S1: Add stage section for a the flow elements
**Description**
Add Stage Section showed only when the element selected is a FLOW

gproly committed:
FOUR-23473
 ##Description
Add translate
https://processmaker.atlassian.net/browse/FOUR-23473

pmPaulis committed:
Merge pull request #8204 from ProcessMaker/feature/FOUR-23575

FOUR-22485: Open launchpad buttons appears templates edit

pmPaulis committed:
Merge pull request #8209 from ProcessMaker/feature/FOUR-23473

FOUR-23473 S1: Add stage section for a the flow elements

pmPaulis committed:
Merge pull request #8208 from ProcessMaker/feature/FOUR-23484

FOUR-23484: S4:FE-Process Info update & Add the UI component to show the total cases per stage

pmPaulis committed:
Adding stages

henryjonathanquispe committed:
FOUR-23577: S12: Implement component Financial Aid - College Dashboard

henryjonathanquispe committed:
FOUR-23577: S12: Implement component Financial Aid - College Dashboard

pmPaulis committed:
adding columns related

pmPaulis committed:
Merge pull request #8211 from ProcessMaker/feature/FOUR-23577

FOUR-23577: S12: Implement component Financial Aid - College Dashboard

pmPaulis committed:
Update the tables structure

pmPaulis committed:
Update stages

pmPaulis committed:
Update stages

pmPaulis committed:
Add progress information in CasesController

pmPaulis committed:
Save stages per process

pmPaulis committed:
solving name

gproly committed:
FOUR-23474 S1: Add stage component with drag and drop and Save changes

 ## Description:                                                                                         - The user can Add Stages with the button
- The user can drag and drop the stages for define the order
- When the user select a stage the number of the stage needs to show over the flow and the Stage needs to show selected with blue color
- The stage can edit and delete

 ## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-23475
https://processmaker.atlassian.net/browse/FOUR-23474
https://processmaker.atlassian.net/browse/FOUR-23476
https://processmaker.atlassian.net/browse/FOUR-23477

pmPaulis committed:
Merge pull request #8205 from ProcessMaker/feature/FOUR-23478

FOUR-23478: S2: FE - Add new columns and prepare structure in the tables

pmPaulis committed:
Merge pull request #8215 from ProcessMaker/feature/FOUR-23474

FOUR-23474 S1: Add stage component with drag and drop and Save changes

pmPaulis committed:
FOUR-24219

pmPaulis committed:
Merge pull request #8220 from ProcessMaker/feature/FOUR-24219

FOUR-24219: Update the structure of tables: process_request and cases_started and cases_participated

pmPaulis committed:
FOUR-23485

pmPaulis committed:
FOUR-23485 using the stagesSummary in the ProgressBarSection

henryjonathanquispe committed:
FOUR-24086: S12: Settings - Add a select List to define the view/tce-dashboard and save

pmPaulis committed:
Merge pull request #8221 from ProcessMaker/feature/FOUR-23485

FOUR-23485: Create a new API to return total cases per Stage

fagubla committed:
FOUR-23483: Add in Details the Stage and bar

fagubla committed:
Update CasesController.php

pmPaulis committed:
FOUR-24229

pmPaulis committed:
FOUR-24229

pmPaulis committed:
remove a var_dump

pmPaulis committed:
Merge pull request #8222 from ProcessMaker/feature/FOUR-23483

FOUR-23483: Add in Details the Stage and bar

pmPaulis committed:
Merge pull request #8224 from ProcessMaker/feature/FOUR-24229

FOUR-24229: Get information to show status of the stages in the Case Progress Bar control

henryjonathanquispe committed:
FOUR-24086: S12: Settings - Add a select List to define the view/tce-dashboard and save

henryjonathanquispe committed:
FOUR-24086: S12: Settings - Add a select List to define the view/tce-dashboard and save

pmPaulis committed:
Merge pull request #8226 from ProcessMaker/feature/FOUR-24086

FOUR-24086 : S12: Settings - Add a select List to define the view/tce-dashboard and save

henryjonathanquispe committed:
FOUR-24087: S12: Create the view to show a dashboard with the components related to Distribution Bar Fin Aid Student

fagubla committed:
FOUR-24148: UI related Process Info - Carousel (refactor)

fagubla committed:
reset behavior

fagubla committed:
FOUR-24233:The Summary tab needs to show per default

gproly committed:
FOUR-22734 S1: The Publish and save process needs to save the stages
 ##Description
1. Update the process.stages with all stages.
2. When the flow was assigned a Stage needs to show the NUMBER related to the stage.
3. Update the process.bpmn with the pm:config added in the flow, like: pm:config="{stageId:2; stageName:'Review proposal'}".

pmPaulis committed:
Merge pull request #8231 from ProcessMaker/feature/FOUR-24148

FOUR-24148: UI related Process Info - Carousel (refactor)

henryjonathanquispe committed:
FOUR-24087: S12: Create the view to show a dashboard with the components related to Distribution Bar Fin Aid Student

henryjonathanquispe committed:
FOUR-22605: Resolve conflicts

henryjonathanquispe committed:
FOUR-24087: S12: Create the view to show a dashboard with the components related to Distribution Bar Fin Aid Student

pmPaulis committed:
Merge pull request #8225 from ProcessMaker/epic/FOUR-22600

FOUR-22600: [SUMMER 25] Stages configuration (Progress Bar)

pmPaulis committed:
Merge pull request #8236 from ProcessMaker/feature/FOUR-24087

FOUR-24087: S12: Create the view to show a dashboard with the components related to "Distribution Bar Fin Aid Student"

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

henryjonathanquispe committed:
FOUR-24089: S12: Create the view to show a dashboard with the components related to Financial Aid - College Dashboard

pmPaulis committed:
solving conflicts

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8232 from ProcessMaker/feature/FOUR-24233

FOUR-24233:The Summary tab needs to show per default

henryjonathanquispe committed:
FOUR-24089: S12: Create the view to show a dashboard with the components related to Financial Aid - College Dashboard

henryjonathanquispe committed:
FOUR-24089: S12: Create the view to show a dashboard with the components related to Financial Aid - College Dashboard

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-24227: Adding changes in order to get the status: Done, In progress and Pending

henryjonathanquispe committed:
FOUR-24089: S12: Create the view to show a dashboard with the components related to Financial Aid - College Dashboard

fagubla committed:
FOUR-24227: Add in the display screen a New control - Case Progress Bar

henryjonathanquispe committed:
FOUR-24230: S12: Add the Request List using the columns defined

pmPaulis committed:
Merge pull request #8242 from ProcessMaker/feature/FOUR-24227-P

FOUR-24227: Adding changes in order to get the status: Done, In progres, Pending

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-24234: Creating API stages and metrics per process

pmPaulis committed:
Updating the results

pmPaulis committed:
fixing unit test issue

pmPaulis committed:
fixing unit test issue

pmPaulis committed:
fixing unit test issue

pmPaulis committed:
Merge pull request #8244 from ProcessMaker/feature/FOUR-24234

FOUR-24234: Return information for the Stages and Metrics

henryjonathanquispe committed:
Merge branch 'epic/FOUR-22605' of github.com:ProcessMaker/processmaker into feature/FOUR-24089

henryjonathanquispe committed:
FOUR-23689: Cannot read properties of null (reading 'colorScheme') Error with chart without data in Launchpad chart

gproly committed:
FOUR-22734 S1: The Publish and save process needs to save the stages
 ##Description
1. Update the process.stages with all stages.
2. When the flow was assigned a Stage needs to show the NUMBER related to the stage.
3. Update the process.bpmn with the pm:config added in the flow, like: pm:config="{stageId:2; stageName:'Review proposal'}".

henryjonathanquispe committed:
FOUR-24089: S12: Create the view to show a dashboard with the components related to Financial Aid - College Dashboard

pmPaulis committed:
Merge pull request #8239 from ProcessMaker/feature/FOUR-24089

FOUR-24089 : S12: Create the view to show a dashboard with the components related to "Financial Aid - College Dashboard"

pmPaulis committed:
Merge pull request #8243 from ProcessMaker/feature/FOUR-24227

FOUR-24227: Add in the display screen a New control - Case Progress Bar

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

gproly committed:
FOUR-22734 Add endpoints to save and load stages in the modeler, and fix code style.

gproly committed:
FOUR-22734 Fix incorrectly selected stages.

gproly committed:
FOUR-22734 Fix null values for stages.

gproly committed:
FOUR-22734 Fixed the id field to ensure it is unique, and perform the removal of the selected property on a copy instead of the main reference.

pmPaulis committed:
FOUR-24234: Adding information about stages

pmPaulis committed:
Merge pull request #8234 from ProcessMaker/feature/FOUR-22734

FOUR-22734 S1: The Publish and save process needs to save the stages

pmPaulis committed:
Merge remote-tracking branch 'origin/release-2025-spring' into epic/FOUR-22600

pmPaulis committed:
Merge pull request #8247 from ProcessMaker/feature/FOUR-24234-2

FOUR-24234: API Return information for the Stages and Metrics

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-24448

pmPaulis committed:
FOUR-24450

pmPaulis committed:
FOUR-24448: Update default stages in other use cases

pmPaulis committed:
Merge pull request #8251 from ProcessMaker/feature/FOUR-24448

FOUR-24448: Consider use cases for return Default Stages per Case and case_number not required

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-24453: Define the structure to save the aggregations in process, use this value in the calcs

pmPaulis committed:
Merge pull request #8252 from ProcessMaker/feature/FOUR-24450

FOUR-24450: Add indexes to improve the queries about stage_id

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge branch 'epic/FOUR-22605' of github.com:ProcessMaker/processmaker into feature/FOUR-24453

pmPaulis committed:
FOUR-24453: Update using the new structure of stages

pmPaulis committed:
removing unused function

gproly committed:
FOUR-23482 S2: BE - Populate the STAGE and PROGRESS columns in run case execution and show in the cases Lists

gproly committed:
FOUR-23482 Fixed code style.

henryjonathanquispe committed:
FOUR-24454: S12: Update/Adjustments some behaviors the Dashboard

henryjonathanquispe committed:
FOUR-24454: S12: Update/Adjustments some behaviors the Dashboard

fagubla committed:
FOUR-23607:Add a Link to configure cases column & show the component to configure columns

pmPaulis committed:
FOUR-22605: Update label with name

gproly committed:
FOUR-23482 Improvement to allow all flows to be tracked and retrieve the stage values.

gproly committed:
FOUR-23482 Fixed code style issues.

fagubla committed:
Title corrections

gproly committed:
FOUR-23482 Improvement for the calculation of the progress field.

gproly committed:
FOUR-23482 Fixed code style issues.

pmPaulis committed:
Merge pull request #8259 from ProcessMaker/feature/FOUR-23607

FOUR-23607:Add a Link to configure cases column & show the component to configure columns

fagubla committed:
FOUR-24336:Create a new View when a case is open and the TCE_CUSTOMIZATION_ENABLED is enabled

henryjonathanquispe committed:
FOUR-24454: S12: Update/Adjustments some behaviors the Dashboard

gproly committed:
FOUR-23482 Improvements: we no longer use Nayra’s event subscriber.
We now retrieve the configuration directly from the BPMN using the `element_id` field found in the `process_request_tokens` table. A function has been created to get and set this data, along with its corresponding test.

gproly committed:
FOUR-23482 Fixed observation issues.

gproly committed:
FOUR-23482 Fixed observation issues.

pmPaulis committed:
Merge pull request #8265 from ProcessMaker/feature/FOUR-24336

FOUR-24336:Create a new View when a case is open and the TCE_CUSTOMIZATION_ENABLED is enabled

pmPaulis committed:
Solving issue with test

pmPaulis committed:
validating the function

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

gproly committed:
FOUR-23482 Fixed unit test issues.

pmPaulis committed:
Merge pull request #8256 from ProcessMaker/feature/FOUR-23482

FOUR-23482 S2: BE - Populate the STAGE and PROGRESS columns in run ca…

henryjonathanquispe committed:
FOUR-24454: S12: Update/Adjustments some behaviors the Dashboard

pmPaulis committed:
Merge remote-tracking branch 'origin/release-2025-spring' into epic/FOUR-22605

pmPaulis committed:
Merge pull request #8257 from ProcessMaker/feature/FOUR-24454

FOUR-24454 : S12: Update/Adjustments some behaviors the Dashboard

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-22600 and FOUR-22605 merged with conflicts solved

pmPaulis committed:
Merge pull request #8267 from ProcessMaker/epic/FOUR-22600_FOUR-22605

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Solving conflicts

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-24453: API related to aggregation

henryjonathanquispe committed:
FOUR-23609: 3. S3: Display the column configured in My Cases List

henryjonathanquispe committed:
FOUR-23609: 3. S3: Display the column configured in My Cases List

pmPaulis committed:
Merge pull request #8254 from ProcessMaker/feature/FOUR-24453

FOUR-24453: Define the structure to save the aggregations in process, update calculation for aggregation and progress

henryjonathanquispe committed:
FOUR-23609: 3. S3: Display the column configured in My Cases List

henryjonathanquispe committed:
Merge branch 'epic/FOUR-22605' of github.com:ProcessMaker/processmaker into feature/FOUR-23609

henryjonathanquispe committed:
FOUR-23609: 3. S3: Display the column configured in My Cases List

henryjonathanquispe committed:
FOUR-23609: 3. S3: Display the column configured in My Cases List

pmPaulis committed:
FOUR-24456

pmPaulis committed:
Merge pull request #8270 from ProcessMaker/feature/FOUR-23609

Feature/four 23609

henryjonathanquispe committed:
FOUR-23609: 3. S3: Display the column configured in My Cases List

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

fagubla committed:
FOUR-24452: Add a Agreggation property in modeler

pmPaulis committed:
Adding new variable to use metricsApiEndpoint

pmPaulis committed:
Add Progress column

pmPaulis committed:
Merge pull request #8274 from ProcessMaker/feature/FOUR-24452

FOUR-24452: Add a Agreggation property in modeler

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8272 from ProcessMaker/feature/FOUR-24456

FOUR-24456: Define how we can get Metrics information

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

fagubla committed:
FOUR-24554:Sync: Cases - Enable the columns Stage and Progress

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

pmPaulis committed:
Merge pull request #8279 from ProcessMaker/feature/FOUR-24554

FOUR-24554:Sync: Cases - Enable the columns Stage and Progress

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

pmPaulis committed:
FOUR-2463: Error is showing in the API: api/1.0/cases/stages_bar/2

fagubla committed:
FOUR-24580:Process Info - Started Cases and total is not showing

pmPaulis committed:
Merge pull request #8283 from ProcessMaker/feature/FOUR-24636

FOUR-24636: Error is showing in the API: api/1.0/cases/stages_bar/2

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

fagubla committed:
Update ProcessInfo.vue

henryjonathanquispe committed:
FOUR-24474: S12: API request/ needs to consider a filter by Stage_id

pmPaulis committed:
Merge pull request #8277 from ProcessMaker/feature/FOUR-24474

FOUR-24474 S12: API "request/" needs to consider a filter by Stage_id

pmPaulis committed:
FOUR-24457

pmPaulis committed:
Merge pull request #8285 from ProcessMaker/feature/FOUR-24580

FOUR-24580:Process Info - Started Cases and total is not showing

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Solving unit tests

pmPaulis committed:
FOUR-24717: adding key total and stages

pmPaulis committed:
Merge pull request #8292 from ProcessMaker/feature/FOUR-24457

FOUR-2445: S12: API - Return Metrics information

henryjonathanquispe committed:
Solve issues

henryjonathanquispe committed:
FOUR-24717: Update the Total Information for Stages

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
solving observation

henryjonathanquispe committed:
FOUR-24707: Stages - Update the Checklist design with click option

gproly committed:
FOUR-23608 2. S3: Storage the columns into process_launchpad.properties.my_cases_columns

 ## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-23608

pmPaulis committed:
Merge pull request #8295 from ProcessMaker/feature/FOUR-24717

FOUR-24717: Update the Total Information for Stages

henryjonathanquispe committed:
FOUR-24707: Stages - Update the Checklist design with click option

gproly committed:
iFOUR-23608 Read columns from CustomHomeTableSection

henryjonathanquispe committed:
FOUR-24757: UI: Update Grants according the figma

henryjonathanquispe committed:
FOUR-24764: UI: Student and College according figma

pmPaulis committed:
Merge pull request #8297 from ProcessMaker/feature/FOUR-23608

FOUR-23608 2. S3: Storage the columns into process_launchpad.properti…

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8298 from ProcessMaker/bugfix/FOUR-24757

FOUR-24757: UI: Update Grants according the figma

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

gproly committed:
FOUR-24707 Removing unnecessary changes; the required logic is added to remove the checkbox and replace it with click and double-click functionality to select and deselect the stage.

gproly committed:
FOUR-24707 Fix code style.

gproly committed:
FOUR-24707 Fix code style.

gproly committed:
FOUR-24707 Fix code style.

pmPaulis committed:
FOUR-24803

pmPaulis committed:
FOUR-24803

pmPaulis committed:
Merge pull request #8296 from ProcessMaker/bugfix/FOUR-24707

FOUR-24707: Stages - Update the Checklist design with click option

fagubla committed:
FOUR-24705:The Stage Name is required

fagubla committed:
validation

fagubla committed:
validation

pmPaulis committed:
Merge pull request #8321 from ProcessMaker/bugfix/FOUR-24705

FOUR-24705:The Stage Name is required

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

henryjonathanquispe committed:
FOUR-24764: UI: Student and College according figma

pmPaulis committed:
Solving conflicst with develop

fagubla committed:
FOUR-24773:One or two lines should be visible in stages name configuration

pmPaulis committed:
Merge pull request #8328 from ProcessMaker/bugfix/FOUR-24773

FOUR-24773:One or two lines should be visible in stages name configuration

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

fagubla committed:
FOUR-24795:The stage ID is duplicated if the first stages are deleted.

pmPaulis committed:
Merge pull request #8305 from ProcessMaker/feature/FOUR-24803

FOUR-24803

henryjonathanquispe committed:
FOUR-24764: UI: Student and College according figma

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8332 from ProcessMaker/bugfix/FOUR-24795

FOUR-24795:The stage ID is duplicated if the first stages are deleted.

fagubla committed:
FOUR-24890:The FE needs to validate if the API does not exist or parameters

pmPaulis committed:
Merge pull request #8300 from ProcessMaker/bugfix/FOUR-24764

FOUR-24764 UI: Student and College according figma

fagubla committed:
FOUR-24972: There is not scroll in "Completed & Form" Cases

fagubla committed:
add default height

henryjonathanquispe committed:
FOUR-24778: The columns data.someFields does not show the correct information

pmPaulis committed:
Merge pull request #8342 from ProcessMaker/bugfix/FOUR-24778

FOUR-24778: The columns data.someFields does not show the correct information

pmPaulis committed:
FOUR-24887

pmPaulis committed:
Merge pull request #8340 from ProcessMaker/bugfix/FOUR-24972

FOUR-24972: There is not scroll in "Completed & Form" Cases

pmPaulis committed:
FOUR-25809

pmPaulis committed:
Merge pull request #8336 from ProcessMaker/bugfix/FOUR-24890

FOUR-24890:The FE needs to validate if the API does not exist or parameters

fagubla committed:
validation response

pmPaulis committed:
Merge pull request #8344 from ProcessMaker/feature/FOUR-25089

FOUR-25809: Columns in cases are not matched from launchpad configuration

pmPaulis committed:
Solving issue with show the stage

pmPaulis committed:
FOUR-24981

pmPaulis committed:
Update translations

pmPaulis committed:
Revert the change in the public

fagubla committed:
FROU-24983:It is possible to add more than 8 stages in modeler

fagubla committed:
correction

pmPaulis committed:
Merge pull request #8343 from ProcessMaker/feature/FOUR-24887

FOUR-24887: The Cases list show and empty Stage Columns when the process does not have Stages

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

henryjonathanquispe committed:
FOUR-24891 : FE: remove the fake variables related to the custom columns

pmPaulis committed:
Merge pull request #8348 from ProcessMaker/bugfix/FOUR-24983

FOUR-24983: It is possible to add more than 8 stages in modeler

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8345 from ProcessMaker/bugfix/FOUR-24890

FOUR-24890: validation response

pmPaulis committed:
Merge pull request #8350 from ProcessMaker/bugfix/FOUR-24891

FOUR-24891 : FE: remove the fake variables related to the custom columns

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Solving issue in the Create stage

pmPaulis committed:
adding code styles

pmPaulis committed:
adding translations

pmPaulis committed:
Merge pull request #8347 from ProcessMaker/feature/FOUR-24981

FOUR-24981: It is not possible to start a case if the process has large stage name

pmPaulis committed:
Merge remote-tracking branch 'origin/develop' into epic/FOUR-22605

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge remote-tracking branch 'origin/develop' into epic/FOUR-22605

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-25271

pmPaulis committed:
FOUR-24887

pmPaulis committed:
Merge pull request #8360 from ProcessMaker/feature/FOUR-25271

FOUR-25271: Total cases does not match with cases started in launchpad to Grant chart

pmPaulis committed:
Merge pull request #8365 from ProcessMaker/feature/FOUR-25306

FOUR-24887: UI: the progress Bar needs work when the stages are the defaults

henryjonathanquispe committed:
FOUR-25222: There is no synchronization with the 'i' button and the form process info when it is open or closed

pmPaulis committed:
Merge pull request #8376 from ProcessMaker/bugfix/FOUR-25222

FOUR-25222: There is no synchronization with the 'i' button and the form process info when it is open or closed

pmPaulis committed:
solving conflicts with develop

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

henryjonathanquispe committed:
FOUR-25269: In Grant chart the tabs In progress and Completed does not listed any request

pmPaulis committed:
Fixing unit test

pmPaulis committed:
Merge pull request #8377 from ProcessMaker/bugfix/FOUR-25269

FOUR-25269: In Grant chart the tabs In progress and Completed does not listed any request

pmPaulis committed:
FOUR-24578

pmPaulis committed:
solving observations

henryjonathanquispe committed:
FOUR-25283: It is not possible to change other variable in amount

pmPaulis committed:
Merge pull request #8378 from ProcessMaker/bugfix/FOUR-24578

FOUR-24578: The Stage for the End Event is not registered and the progress is not showing correctly

pmPaulis committed:
Merge pull request #8386 from ProcessMaker/bugfix/FOUR-25283

FOUR-25283: It is not possible to change other variable in amount

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

henryjonathanquispe committed:
FOUR-25355 : Move Progress Column in Cases List

pmPaulis committed:
Merge pull request #8392 from ProcessMaker/bugfix/FOUR-25355

FOUR-25355 : Move Progress Column in Cases List

henryjonathanquispe committed:
FOUR-25426:UI: There is no validation of variables in Stage Aggregation

fagubla committed:
FOUR-25273:Labels in stage flows are not lost after deleting all stages.

henryjonathanquispe committed:
FOUR-25356: UI: Review the render of TCE Dasboards

henryjonathanquispe committed:
FOUR-25426:UI: There is no validation of variables in Stage Aggregation

henryjonathanquispe committed:
FOUR-25356: UI: Review the render of TCE Dasboards

henryjonathanquispe committed:
FOUR-25356: UI: Review the render of TCE Dasboards

henryjonathanquispe committed:
FOUR-25356: UI: Review the render of TCE Dasboards

pmPaulis committed:
Merge pull request #8397 from ProcessMaker/bugfix/FOUR-25273

FOUR-25273: Labels in stage flows are not lost after deleting all stages.

henryjonathanquispe committed:
FOUR-25282: UI: Cards in screen launcher are not the same as in figma when this calculate the Amount

pmPaulis committed:
Merge pull request #8396 from ProcessMaker/bugfix/FOUR-25426

FOUR-25426:UI: There is no validation of variables in Stage Aggregation

pmPaulis committed:
Merge pull request #8401 from ProcessMaker/bugfix/FOUR-25282

FOUR-25282: UI: Cards in screen launcher are not the same as in figma when this calculate the Amount

fagubla committed:
FOUR-25425:The stage is assigned to the flow when it is edited

pmPaulis committed:
Merge pull request #8399 from ProcessMaker/bugfix/FOUR-25356

FOUR-25356: UI: Review the render of TCE Dasboards

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

gproly committed:
FOUR-25219 Changes are not reflected in "Active Columns" in “Edit Case Columns“
 ## Description:
Changes are not reflected in "Active Columns" in “Edit Case Columns“

 ## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-25219

pmPaulis committed:
Merge pull request #8403 from ProcessMaker/bugfix/FOUR-25425

FOUR-25425:The stage is assigned to the flow when it is edited

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8404 from ProcessMaker/bugfix/FOUR-25219

FOUR-25219 Changes are not reflected in "Active Columns" in “Edit Cas…

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
FOUR-25467

fagubla committed:
FOUR-24579:Update stages according the alternatives

pmPaulis committed:
Merge remote-tracking branch 'origin/develop' into epic/FOUR-22605

pmPaulis committed:
Merge branch 'epic/FOUR-22605' of github.com:ProcessMaker/processmaker into feature/FOUR-25467

pmPaulis committed:
Merge remote-tracking branch 'origin/epic/FOUR-22605' into bugfix/FOUR-24579

pmPaulis committed:
Merge pull request #8414 from ProcessMaker/bugfix/FOUR-24579

FOUR-24579: Update stages according the alternatives

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
Merge pull request #8407 from ProcessMaker/feature/FOUR-25467

FOUR-25467: How we can improve the behavior when some flows does not have stages

vladyrichter commented:
QA server K8S was successfully deployed https://ci-8fbaaa065c.engk8s.processmaker.net

pmPaulis committed:
solving conflict

pmPaulis commented:
Team,
The conflicts was solve.
Regards
